.gitmodules

```diff
@@ -1,39 +1,39 @@
 [submodule "core/include/glm"]
-	path = core/include/glm
+	path = core/deps/glm
```

Comment:
Are we still planning to directly put the _cold_ submodules as files within the repo?